app/v1/book.py

```diff
@@ -0,0 +1,83 @@
+from docopt import DocoptExit, docopt
```

Comment:
- Why have you used docopt? Are you creating a CLI?
- I also see you have commented out the implementation of your functions. Why is that?

Reply:
Thank you @geeeh  , the use of docopt is the first import we were taught on day one of Bootcamp by Mr. George (I forget his second name). He talked about the need to do quick tests. 

About your second Query, in the task we were given on Friday, we needed to research about PEP-8. In it we are told that when writing code, we must remember that code is read more often than it is written. Comments are a responsible way to communicate what we are doing. Code with little or no comments is considered sub-standard by some quarters.

Am looking forward to more reviews Sir.